cmd/bootstrap/cmd/snapshot.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+
+	"github.com/onflow/flow-go-sdk/client"
+
+	model "github.com/onflow/flow-go/model/bootstrap"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/state/protocol/inmem"
+)
+
+var flagAccessAddress string
+var flagNodeID string
+
+// snapshotCmd represents the command to download the latest protocol snapshot to bootstrap from
+var snapshotCmd = &cobra.Command{
+	Use:   "snapshot",
+	Short: "Downloads the latest serialized protocol snapshot from an access node to be used to bootstrap from",
+	Long:  `Downloads the latest serialized protocol snapshot from an access node to be used to bootstrap from`,
+	Run:   snapshot,
+}
+
+func init() {
+	rootCmd.AddCommand(snapshotCmd)
+
+	// required parameters
+	snapshotCmd.Flags().StringVar(&flagAccessAddress, "access-address", "", "the address of an access node")
+	_ = snapshotCmd.MarkFlagRequired("access-address")
+	snapshotCmd.Flags().StringVar(&flagNodeID, "node-id", "", "")
+	_ = snapshotCmd.MarkFlagRequired("access-address")
```

Comment:
We can assume this command is being run with a reference to the bootstrap directory as in the `transit` script and can get the node ID like:
https://github.com/onflow/flow-go/blob/cf0c6679ff20d3dbce989876830d8deb2fa9e4ed/cmd/bootstrap/transit/main.go#L135-L144